definitions/infra-azureservicebus/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: INFRA
+type: AZURESERVICEBUS
+goldenTags:
+- azure.regionName
+- azure.subscriptionId
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
+synthesis:
+  rules:
+  - identifier: azure.resourceId
+    name: displayName
+    legacyFeatures:
+      overrideGuidType: true
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: azure.resourceType
+      value: microsoft.servicebus/namespaces
```

Comment:
There is a rule equal to this one already in the repo, should we choose between `INFRA-AZURESERVICEBUS` and `INFRA-AZURESERVICEBUSNAMESPACE`?

<img width="1124" alt="Screenshot 2023-05-12 at 15 04 32" src="https://github.com/newrelic/entity-definitions/assets/611147/7a124564-cf4a-4b85-9f78-53b613b33d07">
<img width="1565" alt="Screenshot 2023-05-12 at 15 04 23" src="https://github.com/newrelic/entity-definitions/assets/611147/8ff0bedc-cb14-41a0-bfb3-5b4ba979baec">

Reply:
@glennthomas1 assumed you want to remove the old one and removed it here: https://github.com/newrelic/entity-definitions/pull/1067/commits/b490f263e8adef01b59d8e4a8001747e0bf3e54c

Please let us know and we can merge it after that.